fe/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -284,7 +286,6 @@ public void clearExportStatus() {
         lock.lock();
         try {
             this.backendExecStates.clear();
```

Comment:
needCheckBackendExecStates is need to be cleared?

Reply:
Yes. I fix it